springboot/springboot.bzl

```diff
@@ -385,6 +385,7 @@ def springboot(
         name,
         java_library,
         boot_app_class,
+        spring_boot_launcher_class = "org.springframework.boot.loader.JarLauncher",
```

Comment:
nitpick: for consistency, can you rename attr to boot_launcher_class

Reply:
Of course